.github/workflows/stale.yml

```
@@ -0,0 +1,25 @@
+  push:
+    branches: [ master, development ]
+  pull_request:
+    branches: [ master, development ]
+
+jobs:
+  stale:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/stale@v1
```

actionlint (1.7.7)

the runner of "actions/stale@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)